### PR TITLE
PR : header and footer layout

### PR DESCRIPTION
**Pull request by shailja-imw**
=>Header design added
=>Footer design

### CODE REVIEW

**Comment by shailja-imw** [comment]
**Layout Header and Footer**

image

File: app/app.vue [hunk 1]
```diff
@@ -1,3 +1,9 @@
 <template>
-  <NuxtPage />
+  <div>
+    <NuxtLayout>
+      <div class="container">
```
Comment (monikagorkhiya) [inline app/app.vue:4]: App.vue you can remove also , in the layout you can directly get the pages. 
but This way is also proper
i think container should be use section wise

File: app/layouts/default.vue [hunk 2]
```diff
@@ -0,0 +1,114 @@
...
+                >
+              </li>
+            </ul>
+          </form>
+        </div>
+      </div>
+    </nav>
+    <!-- Nav bar ended -->
+    <div class="main-content">
+      <slot></slot>
```
Comment (monikagorkhiya) [inline app/layouts/default.vue:65]: here you can use NuxtPage

**Review by monikagorkhiya — CHANGES_REQUESTED** [review]

File: app/app.vue (hunk 1 above)
Comment (shailja-imw) [reply app/app.vue:4]: Ok, So I will remove the app.vue and in index, I will use <NuxtPage> .

File: app/assets/css/main.css [hunk 3]
```diff
@@ -0,0 +1,84 @@
+body {
```
Comment (Payal17122000) [inline app/assets/css/main.css:1]: not use direct style for tag

File: app/assets/css/main.css [hunk 4]
```diff
@@ -0,0 +1,84 @@
...
+  background-color: #151f30;
+  color: #fff;
+  min-height: 100vh;
+  display: flex;
+  flex-direction: column;
+}
+
+a {
```
Comment (Payal17122000) [inline app/assets/css/main.css:9]: again not use direct style to tag

**Review by Payal17122000 — CHANGES_REQUESTED** [review]

**Review by Payal17122000 — APPROVED** [review]

File: app/layouts/default.vue [hunk 5]
```diff
@@ -0,0 +1,119 @@
...
+                  to="/discover"
+                  >Discover</NuxtLink
+                >
+              </li>
+              <li class="nav-item px-2">
+                <NuxtLink
+                  class="nav-link white-text-link"
+                  aria-current="page"
+                  to="/add"
+                  >Add Show</NuxtLink
```
Comment (monikagorkhiya) [inline app/layouts/default.vue:60]: Please add this as a btn in Discover page

**Review by monikagorkhiya — CHANGES_REQUESTED** [review]

File: app/layouts/default.vue (hunk 5 above)
Comment (shailja-imw) [reply app/layouts/default.vue:60]: Add show need to replace from here to discover page?

**Review by monikagorkhiya — APPROVED** [review]

**Pull request merged**